### PR TITLE
Kris

### PR DESCRIPTION
**Pull request by kristi-h**

### CODE REVIEW

**Review by belunatic — APPROVED** [review]

**Review by ccespedes — APPROVED** [review]

File: expense-splitter/src/components/group/CreateGroup.jsx
Comment (singhalex) [inline expense-splitter/src/components/group/CreateGroup.jsx:9]: I'm getting an error here. setGroupData is passed in as props, but also declared on line 12. Does it need to be passed it or is it just pulling in from context?

File: expense-splitter/src/components/ui/MultiSelectDropdown.jsx
Comment (singhalex) [inline expense-splitter/src/components/ui/MultiSelectDropdown.jsx:1]: Is it possible to have the dropdown close whenever you click outside of it? As it stands now, you have to click the select friends button again to close it.

**Review by singhalex — CHANGES_REQUESTED** [review]
Hello Kris, I identified an error that needs to be addressed before approval and also a suggestion on the new dropdown ui component

Thanks

File: expense-splitter/src/components/ui/MultiSelectDropdown.jsx
Comment (ccespedes) [reply expense-splitter/src/components/ui/MultiSelectDropdown.jsx:1]: Yes, this is something we noted as well. I was going to tackle that. We're still kinda fiddling around with it.

File: expense-splitter/src/components/group/CreateGroup.jsx
Comment (ccespedes) [reply expense-splitter/src/components/group/CreateGroup.jsx:9]: I merged it before seeing this, sorry about that Alex.

**Pull request merged**
